### ISSUE TITLE
[develop branch] Radial menu sometimes incorrectly sets mouse position

### ISSUE THREAD

**Issue by OctothorpeObelus** [CapsAdmin/pac3#1342]
Seemingly at random, opening the radial events menu will set the mouse position to the right side of the wheel rather than in the center. Latest commit in use. To reproduce, use the develop branch and simply use the event wheel a few times. It does not appear to be consistent but it does happen often.

![image](https://github.com/CapsAdmin/pac3/assets/10675391/047fc351-bb5e-47b2-a908-5c2ab9dcfa8b)

**Comment by pingu7867** [comment]
Where's your cursor in the screenshot? I don't see it.

Our eventwheel code doesn't set cursor pos. gmod constantly sets it to the center when in first person, but you have stuff like the C context menu that saves its own cursor position.

The problem is the gui.EnableScreenClicker mode doesn't know which menu element, game mode or hacky GUI render hook you're supposed to be on or which one you want to use. Switching has to be done manually and we have to hope there's no conflicting codes that fight over control of it because each mode change forces its gui.EnableScreenClicker state over the whole game.

The way the original eventwheel works doesn't focus on how far the mouse is relative to the center, but rather it does angle checks. But if your cursor is smack dab at the center, it defaults to a rightward angle. When gui.EnableScreenClicker is turned off in the middle of when it's shown, the cursor reverts to gmod's default behavior, it gets placed at the center and the event wheel thinks it's "right".

TL;DR you have something else running gui.EnableScreenClicker(false) before you closed the eventwheel. That might be the eventlist (the static grid variation of the eventwheel) or another addon.
